### PR TITLE
Q3.txt

### PR DESCRIPTION
**Pull request by NoorAlkattan**

### CODE REVIEW

File: HW10/read.txt [hunk 1]
```diff
@@ -0,0 +1,101 @@
+var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
```
Comment (ssharba) [inline HW10/read.txt]: @NoorAlkattan can we do this with one loop?

**Review by ssharba — CHANGES_REQUESTED** [review]

File: HW10/read.txt (hunk 1 above)
Comment (NoorAlkattan) [reply HW10/read.txt]: Hi Samer , what do you mean by one loop?

Comment (ssharba) [reply HW10/read.txt]: you are looping through the array for each condition, I would start one loop and combine all if conditions inside it.

**Review by ssharba — CHANGES_REQUESTED** [review]

File: HW10/read.txt (hunk 1 above)
Comment (NoorAlkattan) [reply HW10/read.txt]: var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];

 var i=0;
 var x=0;

 for (i=0;i<b.length;i++){
    if(b[i]%2 !==0){
        console.log(b[i]*2);
    }
        if (b[i]%2 ===0)
        console.log(b[i]-2);

```
 if(b[i]%5 === 0){
 console.log(Math.pow(b[i],2));
 }
 if(b[i]%10 === 0){
for (x=0; x<=b[i];x+=10){
    console.log (x);
}
 }
```

 }

Comment (NoorAlkattan) [reply HW10/read.txt]: please check see if that what exactly you want , so i can correct Ruby

Comment (ssharba) [reply HW10/read.txt]: yes

Comment (NoorAlkattan) [reply HW10/read.txt]: so its right , do you want me to correct ruby also and push it again through github

**Comment by ssharba** [comment]
yes, please.

**Pull request closed without merging**